### PR TITLE
fix: Add loading and countdown status to the verification code sending button

### PR DESCRIPTION
**Pull request by jianmaikj**
A(R73B CLH)`GWUWN 4UQ5
W%UI0D8(3T608IX XO)5SP7

### CODE REVIEW

**Comment by Steve0x2a** [comment]
This pr looks good, but seems to contain code for https://github.com/casbin/casdoor/pull/386

**Comment by solo-xwz** [comment]
> This pr looks good, but seems to contain code for #386

Indeed, I reverted that changes

**Comment by jianmaikj** [comment]
> This pr looks good, but seems to contain code for #386

Indeed, I reverted that changes

**Review by Steve0x2a — COMMENTED** [review]
lgtm

**Comment by seriouszyx** [comment]
Please fix. 

- [You have 3 commits incorrectly signed off.](https://github.com/casbin/casdoor/pull/384/checks?check_run_id=4594343165)
- Rename the PR title and commit message with conventions such as [Angular](https://www.conventionalcommits.org/en/v1.0.0/#examples).

image

File: web/src/component/CountDownInput.js [hunk 1]
```diff
@@ -81,22 +89,23 @@ export const CountDownInput = (props) => {
...
-            borderRadius: "3px",
-            border: "1px solid #ccc",
-            marginBottom: 10
-          }}
-        />
-        <Row>
-          <Input autoFocus value={key} prefix={<SafetyOutlined />} placeholder={i18next.t("general:Captcha")} onPressEnter={handleOk} onChange={e => setKey(e.target.value)} />
-        </Row>
-      </Col>
+        <Col>
```
Comment (hsluoyz) [inline web/src/component/CountDownInput.js:92]: Don't change the 2-space indentation.

File: web/src/component/CountDownInput.js [hunk 2]
```diff
@@ -106,39 +115,38 @@ export const CountDownInput = (props) => {
...
-        width={248}
-      >
-        {
-          renderCheck()
-        }
-      </Modal>
-    </React.Fragment>
+      <React.Fragment>
+        <Search
+            addonBefore={textBefore}
```
Comment (hsluoyz) [inline web/src/component/CountDownInput.js:120]: Don't change the 2-space indentation.

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: web/src/component/CountDownInput.js (hunk 2 above)
Comment (jianmaikj) [reply web/src/component/CountDownInput.js]: fixed

**Review by hsluoyz — APPROVED** [review]

**Comment by hsluoyz** [comment]
@jianmaikj plz fix:

image

**Pull request merged**